docs/antctl.md

```diff
@@ -152,6 +152,8 @@ antctl get agentinfo
 
 Both Antrea Controller and Agent support querying NetworkPolicy objects.
 
+The NetworkPolicy here means the control plane NetworkPolicy, instead of its source NetworkPolicy. The source NetworkPolicy of a control plane NetworkPolicy is the original policy resource (K8s NetworkPolicy or Antrea-native Policy) from which the control plane NetworkPolicy was derived.
```

Comment:
How about:
"Both Antrea Controller and Agent support querying the NetworkPolicy objects in the Antrea control plane API. The source of a control plane NetworkPolicy...".

"The source NetworkPolicy of" -> "The source of".

Reply:
Good one.